docs/src/main/asciidoc/security-proactive-authentication.adoc

```diff
@@ -13,36 +13,44 @@ Proactive authentication is enabled in Quarkus by default. This means that if an
-You can then subscribe to the resulting `Uni` and will be notified when authentication is complete and the identity is available.
+Then, you can subscribe to the resulting `Uni` and will be notified when authentication is complete and the identity is available.
 
-NOTE: It's still possible to access the `SecurityIdentity` synchronously with `public SecurityIdentity getIdentity()` in the xref:resteasy-reactive.adoc[RESTEasy Reactive] from endpoints annotated with `@RolesAllowed`, `@Authenticated`, or with respective configuration authorization checks as authentication has already happened.
-The same is also valid for the xref:reactive-routes.adoc[Reactive routes] if a route response is synchronous.
+[NOTE]
+====
+You can still access `SecurityIdentity` synchronously with `public SecurityIdentity getIdentity()` in xref:resteasy-reactive.adoc[RESTEasy Reactive] from endpoints that are annotated with `@RolesAllowed`, `@Authenticated`, or with respective configuration authorization checks because authentication has already happened.
+The same is also valid for xref:reactive-routes.adoc[Reactive routes] if a route response is synchronous.
+====
 
-xref:security-authorization.adoc#standard-security-annotations[Standard security annotations] on CDI beans are not supported on IO thread if a non-void secured method returns a value synchronously and proactive authentication is disabled, as they need to access the `SecurityIdentity`.
-In the example below, we have defined `HelloResource` and `HelloService`. It's easy to see that any GET request to `/hello` will run on IO thread and throw `BlockingOperationNotAllowedException` exception.
+xref:security-authorization.adoc#standard-security-annotations[Standard security annotations] on CDI beans are not supported on an I/O thread if a non-void secured method returns a value synchronously and proactive authentication is disabled because they need to access `SecurityIdentity`.
+
+In the following example, `HelloResource` and `HelloService` are defined.
+Any GET request to `/hello` will run on the I/O thread and throw a `BlockingOperationNotAllowedException` exception.
 
 There is more than one way to fix the example:
 
-* switch to a worker thread (annotate `hello` endpoint with `@Blocking`)
-* change `sayHello` method return type (use reactive or asynchronous data type)
-* arguably the safest way is to move `@RolesAllowed` annotation to the endpoint, as accessing `SecurityIdentity` from endpoint methods is never the blocking operation
+* Switch to a worker thread by annotating the `hello` endpoint with `@Blocking`.
+* Change the `sayHello` method return type by using a reactive or asynchronous data type.
+* Move `@RolesAllowed` annotation to the endpoint.
+This could be one of the safest ways because accessing `SecurityIdentity` from endpoint methods is never the blocking operation.
```

Comment:
Hi @michelle-purcell When this sentence is positioned on its own line it is not clear to me which of the options listed above it refers to - may be it is implied it refers to the last option but I'm not be the only one who may not be sure about it.

Reply:
Hey @sberyozkin - Thanks for reviewing this one too! ... In my local HTML transform, this sentence is together in same line as the final item.  The change is just to apply the one-sentence-one-line approach.

![image](https://github.com/quarkusio/quarkus/assets/92924207/d401c808-3148-40d6-a27e-d9dc6ca6f7ef)

Reply:
@michelle-purcell I see, sounds good, thanks